tests/functional/clients/standalone/package_list/test_java.py

```diff
@@ -1,6 +1,6 @@
 # from ['image']['imagedata']['analysis_report']['package_list']
 from .fixtures import java
-from . import path_params, metadata_params, assert_nested_dict_equal
```

Comment:
@wagoodman not sure how this got missed, but since it no longer exists, it was breaking

Reply:
indeed, we also removed it on https://github.com/anchore/anchore-engine/pull/693 . Essentially one PR was removing it in parallel with another PR that was still utilizing the utility function in new code. One of the PR's didn't have the branch up to date with syft-integration before merging.